byteparse/shared/src/test/scala/byteparse/classparse/ClassTests.scala

```diff
@@ -0,0 +1,1180 @@
+          "65 72 43 6C 61 73 73 01 00 0C 49 6E 6E 65 72 43 6C 61 73 73 65 73 " +
+          "01 00 09 43 4F 4E 53 54 5F 56 41 4C 01 00 01 49 01 00 0D 43 6F 6E " +
+          "73 74 61 6E 74 56 61 6C 75 65 03 00 00 00 2A 01 00 06 3C 69 6E 69 " +
+          "74 3E 01 00 03 28 29 56 01 00 04 43 6F 64 65 01 00 0F 4C 69 6E 65 " +
+          "4E 75 6D 62 65 72 54 61 62 6C 65 01 00 06 6D 65 74 68 6F 64 01 00 " +
+          "14 28 29 4C 6A 61 76 61 2F 6C 61 6E 67 2F 4F 62 6A 65 63 74 3B 01 " +
+          "00 0A 45 78 63 65 70 74 69 6F 6E 73 01 00 0A 44 65 70 72 65 63 61 " +
+          "74 65 64 01 00 09 53 69 67 6E 61 74 75 72 65 01 00 05 28 29 54 54 " +
+          "3B 01 00 19 52 75 6E 74 69 6D 65 56 69 73 69 62 6C 65 41 6E 6E 6F " +
+          "74 61 74 69 6F 6E 73 01 00 16 4C 6A 61 76 61 2F 6C 61 6E 67 2F 44 " +
+          "65 70 72 65 63 61 74 65 64 3B 01 00 28 3C 54 3A 4C 6A 61 76 61 2F " +
+          "6C 61 6E 67 2F 4F 62 6A 65 63 74 3B 3E 4C 6A 61 76 61 2F 6C 61 6E " +
+          "67 2F 4F 62 6A 65 63 74 3B 01 00 0A 53 6F 75 72 63 65 46 69 6C 65 " +
+          "01 00 12 41 74 74 72 69 62 75 74 65 54 65 73 74 2E 6A 61 76 61 0C " +
+          "00 0F 00 10 01 00 18 41 74 74 72 69 62 75 74 65 54 65 73 74 24 49 " +
+          "6E 6E 65 72 43 6C 61 73 73 0C 00 0F 00 25 01 00 0D 41 74 74 72 69 " +
+          "62 75 74 65 54 65 73 74 0C 00 26 00 0C 01 00 13 6A 61 76 61 2F 69 " +
+          "6F 2F 49 4F 45 78 63 65 70 74 69 6F 6E 01 00 10 6A 61 76 61 2F 6C " +
+          "61 6E 67 2F 4F 62 6A 65 63 74 01 00 12 28 4C 41 74 74 72 69 62 75 " +
+          "74 65 54 65 73 74 3B 29 56 01 00 0A 69 6E 6E 65 72 46 69 65 6C 64 " +
+          "00 21 00 04 00 08 00 00 00 01 00 19 00 0B 00 0C 00 01 00 0D 00 00 " +
+          "00 02 00 0E 00 02 00 01 00 0F 00 10 00 01 00 11 00 00 00 1D 00 01 " +
+          "00 01 00 00 00 05 2A B7 00 01 B1 00 00 00 01 00 12 00 00 00 06 00 " +
+          "01 00 00 00 03 00 01 00 13 00 14 00 05 00 11 00 00 00 37 00 03 00 " +
+          "02 00 00 00 17 BB 00 02 59 2A B7 00 03 4C 2B 10 2A B5 00 05 BB 00 " +
+          "06 59 B7 00 07 BF 00 00 00 01 00 12 00 00 00 0E 00 03 00 00 00 0C " +
+          "00 09 00 0D 00 0F 00 0E 00 15 00 00 00 04 00 01 00 06 00 16 00 00 " +
+          "00 00 00 17 00 00 00 02 00 18 00 19 00 00 00 06 00 01 00 1A 00 00 " +
+          "00 03 00 17 00 00 00 02 00 1B 00 1C 00 00 00 02 00 1D 00 0A 00 00 " +
+          "00 0A 00 01 00 02 00 04 00 09 00 01")
+        /* AttributeTest.class from compiled class AttributeTest.java */
+
+
+        val Parsed.Success(parsedClassInfo, _) = ClassParser.classFile.parse(classFile)
+        val parsedClass = ClassParser.Ast.convertToAst(parsedClassInfo)
+
+        assert(parsedClass.fields == ArrayBuffer(
+          Field(
+            "CONST_VAL",
+            "I",
+            FieldFlags(true, false, false, true, true, false, false, false, false),
+            ArrayBuffer(ConstantValueAttribute(IntElem(42))))))
```

Comment:
Could we format the closing parentheses on new lines, similar to the rest of the code e.g. https://github.com/lihaoyi/fastparse/blob/master/pythonparse/shared/src/test/scala/pythonparse/UnitTests.scala#L80-L89 and https://github.com/lihaoyi/fastparse/blob/master/pythonparse/shared/src/test/scala/pythonparse/UnitTests.scala#L310-L315? Good to keep things consistent. Someday scalafmt will automate all this but for now we should do it manually